contrib/keyd/patches/no-systemd.patch

```diff
@@ -0,0 +1,15 @@
+--- a/Makefile
++++ b/Makefile
```

Comment:
you should patch out the -O3 in https://github.com/rvaiya/keyd/blob/eeea96aa4dda2b94a1431362c781064543625246/Makefile#L37

Reply:
thanks. changed this.